src/app/education/education.component.css

```diff
@@ -0,0 +1,221 @@
+.main-container {
+  padding: 40px;
+  background-color: white;
+  border: 1px solid #dedede;
+  border-color: rgba(NaN, 222, 222, 0.5);
+}
+
+.head-container {
+  display: flex;
+}
+
+h1 {
+  text-transform: uppercase;
+  color: #5468ff;
+  font-size: 40px;
+  font-weight: 600;
+  line-height: normal;
+  letter-spacing: 1px;
+  margin-bottom: 30px;
+}
+
+.description-container {
+  width: 620px;
+}
+
+.title {
+  text-transform: uppercase;
+  font-size: 20px;
+  font-weight: 400;
+  color: #484848;
+}
+
+.divider1 {
+  width: 35px;
+  background-color: #5468ff;
+  padding: 1px;
+  margin-top: -5px;
+  margin-bottom: 20px;
+}
+
+mat-divider {
+  margin-top: 30px;
+  margin-bottom: 50px;
+}
+
+img {
+  width: 300px;
+  height: 350px;
+}
+
+.text1 {
```

Comment:
Could you change the name pls to something more descriptive? I helps to better maintain the code.
e.g. header-paragraph maybe?

Reply:
Done